study-builder/studies/mbc/snippets/common/picklist-question-country-required.conf

```diff
@@ -0,0 +1,6579 @@
+{
+  "stableId": "COUNTRY",
+  "questionType": "PICKLIST",
+  "selectMode": "SINGLE",
+  "renderMode": "DROPDOWN",
+  "isRestricted": false,
+  "isDeprecated": false,
+  "hideNumber": false,
+  "promptTemplate": {
```

Comment:
If we don't have to translate the country names, maybe we can just `include` the shared file and override prompt/validations/label with MBC translations.

Reply:
I believe we'll need to translate countries. Hopefully we can reuse translation of countries in future studies.